libunit/tests/ft_isint/00_launcher.c

```diff
@@ -0,0 +1,36 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   00_launcher.c                                      :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: jkhasiza <jkhasiza@student.42.fr>          +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2024/01/20 22:19:43 by jkhasiza          #+#    #+#             */
+/*   Updated: 2024/01/20 23:29:27 by jkhasiza         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "ft_isint_test.h"
+
+void    ft_isint_launcher(void)
+{
+    t_testlist *new;
+    t_testlist *testlist;
+    char        *name;
+
+    testlist = NULL;
+    ft_printf("[FT_ISINT]\n");
+    name = ft_strdup("All chars are digits.");
+    new = testlist_new(name, &test_all_chars_are_digits);
+    if (!new)
+        printf("Failed to create new testlist node\n");
+    testlist_add_back(&testlist, new);
+
+    name = ft_strdup("One character is letter.");
```

Comment:
Name should be protected. Or it shouldn't be allocated and freed in `clear` function.

Reply:
Fixed it.